apps/ledger-live-desktop/src/config/configInit.ts

```diff
@@ -0,0 +1,10 @@
+import { LiveConfig } from "@ledgerhq/live-config/LiveConfig";
+import liveConfigSchema from "@ledgerhq/live-common/config/sharedConfig";
+
+LiveConfig.init({
+  appVersion: __APP_VERSION__,
+  platform: "desktop",
+  environment: process.env.NODE_ENV || "developement",
```

Comment:
shouldn't this be development ? (I didn't check node_env value)

Reply:
I am not following what you meant ?
process.env.NODE_ENV can be "test", "development" or "production".

Reply:
yes, you wrote "developement" here :p

Reply:
very good catch :P 
fixed now